packages/portal-proto/src/features/user-flow/workflow/AnalysisCard.tsx

```diff
@@ -55,68 +55,62 @@ const AnalysisCard: React.FC<AnalysisCardProps> = ({
               query: { app: entry.id },
             }}
             passHref
+            data-testid={`button-${entry.name}`}
+            className={`
+              flex
+              justify-center
+              items-center
+              bg-secondary
+              hover:bg-secondary-dark
+              hover:border-secondary-dark
+              focus:bg-secondary-dark
+              focus:border-secondary-dark
+              mb-1
+              w-[50px]
+              ${inactive ? "opacity-50 pointer-events-none" : ""}
+              rounded
+              h-5
+            `}
+            aria-disabled={inactive}
+            aria-label={entry.name}
```

Comment:
This looks like more than just a nextjs migration.  Can you describe what you are doing here?

Reply:
This was also the result of codemod. What happened here is that the `className`, `data-testid`, `aria-disabled` and `aria-label` which were part of the `a` tag were moved over to the `Link` tag.

Reply:
oh.  i see now.  github did a confusing diff.  i looked at the code in develop and can see what's going on here.  look good.